src/parts/Sidebar/SidebarContent/TestnetBadge/index.tsx

```diff
@@ -5,7 +5,7 @@ import clsx from 'clsx';
 import InterlayCinnabarBadge, { Props as InterlayCinnabarBadgeProps } from 'components/badges/InterlayCinnabarBadge';
 import InterlayTooltip from 'components/UI/InterlayTooltip';
 
-const TestNetBadge = ({
+const TestnetBadge = ({
```

Comment:
Although it's out of scope, I just came across this and wanted to update it according to the industry convention.